test/bdd/features/introduce.feature

```diff
@@ -10,284 +10,177 @@ Feature: Introduce protocol
     Given   "Bob,Carol" exchange DIDs with "Alice"
-      And   "Bob" sends introduce request to the "Alice" asking about "Carol"
-      And   "Alice" sends introduce proposal back to the requester with pub invitation
+    And   "Bob" sends introduce request to the "Alice" asking about "Carol"
+    And   "Alice" sends introduce proposal back to the requester with pub invitation
 
     When   "Bob" wants to know "Carol" and sends introduce response with approve
-    And   "Alice" forwards Carol's invitation to Bob
-
-    Then   "Bob" and "Carol" exchange DIDs
 
     Then   "Alice" checks the history of introduce protocol events "arranging,arranging,arranging,arranging,delivering,delivering,done,done"
-      And   "Bob" checks the history of introduce protocol events "requesting,requesting,deciding,deciding,waiting,waiting,done,done"
+    And   "Bob" checks the history of introduce protocol events "requesting,requesting,deciding,deciding,waiting,waiting,done,done" and stop
 
   @skip_proposal_stop
   Scenario: Alice has a Carol's public invitation but Bob does not want the introduction
     Given   "Bob,Carol" exchange DIDs with "Alice"
-      And   "Alice" sends introduce proposal to the "Bob" with "Carol" invitation
+    And   "Alice" sends introduce proposal to the "Bob" with "Carol" invitation
 
     When   "Bob" doesn't want to know "Carol" and sends introduce response
-      And   "Alice" continue with the introduce protocol
 
-    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,arranging,arranging,delivering,delivering,abandoning,abandoning,done,done"
-      And   "Bob" checks the history of introduce protocol events "deciding,deciding,abandoning,abandoning,done,done"
+    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,arranging,arranging,abandoning,abandoning,done,done"
+    And   "Bob" checks the history of introduce protocol events "deciding,deciding,abandoning,abandoning,done,done" and stop
 
   @skip_proposal_stop_with_request
   Scenario: Alice has a Carol's public invitation but Bob does not want the introduction. The protocol starts with request.
     Given   "Bob,Carol" exchange DIDs with "Alice"
-      And   "Bob" sends introduce request to the "Alice" asking about "Carol"
+    And   "Bob" sends introduce request to the "Alice" asking about "Carol"
 
     When   "Alice" sends introduce proposal back to the requester with pub invitation
-      And   "Bob" doesn't want to know "Carol" and sends introduce response
-
-    Then   "Alice" continue with the introduce protocol
-
-    Then   "Alice" checks the history of introduce protocol events "arranging,arranging,arranging,arranging,delivering,delivering,abandoning,abandoning,done,done"
-      And   "Bob" checks the history of introduce protocol events "requesting,requesting,deciding,deciding,abandoning,abandoning,done,done"
-
-  @skip_proposal_introducer_stop
```

Comment:
is this case no longer relevant?

Reply:
No, since I have deleted the action event after receiving a response it is not possible anymore.